meta-networking/recipes-connectivity/autossh/autossh_1.4g.bb

```diff
@@ -0,0 +1,30 @@
+DESCRIPTION = "autossh is a program to start a copy of ssh and monitor it, restarting it as necessary should it die or stop passing traffic"
+HOMEPAGE = "https://www.harding.motd.ca/autossh/"
+
+LICENSE = "BSD"
+LIC_FILES_CHKSUM = "file://autossh.spec;md5=80a6701134723fd3420e733b46a0eb97"
+
+SRC_URI = "https://www.harding.motd.ca/autossh/autossh-1.4g.tgz"
+SRC_URI[md5sum] = "2b804bc1bf6d2f2afaa526d02df7c0a2"
+SRC_URI[sha256sum] = "5fc3cee3361ca1615af862364c480593171d0c54ec156de79fc421e31ae21277"
+
+RDEPENDS_${PN} = "openssh"
+
+CFLAGS_prepend = "-I${WORKDIR}/build"
```

Comment:
there should be a space after first "

Reply:
It's prepend not append, so it should be space before last ". Or changed to append with leading space.